suite-native/device/src/hooks/useConnectDeviceHandler.tsx

```diff
@@ -30,11 +34,14 @@ export const useConnectDeviceHandler = () => {
     const isIosWithBluetoothEnabled = Platform.OS === 'ios' && isBluetoothEnabled;
 
     const isDeviceThpRequired = useSelector(selectIsDeviceThpRequired);
+    const isAnyPhysicalDeviceConnectedViaUsb = useSelector(
+        selectIsAnyPhysicalDeviceConnectedViaUsb,
+    );
 
     const onConnectDevicePress = useCallback(() => {
         if (isDeviceThpRequired) {
             dispatch(acquireDevice({}));
-        } else if (isIosWithBluetoothEnabled) {
+        } else if (isAnyPhysicalDeviceConnectedViaUsb || isIosWithBluetoothEnabled) {
```

Comment:
Comment taken from PR description. It's not very clear why this would be the case so I would specify that it's a product decision, but feel free to resolve if you find the comment redundant. 

```suggestion
		// In case any device is connected via USB, navigate to the Turn on & unlock screen instead of Connect & unlock.
        } else if (isAnyPhysicalDeviceConnectedViaUsb || isIosWithBluetoothEnabled) {
```

Reply:
What's the added value of a comment that describes exactly what's later done in the code? 🤔

Reply:
Now that I see it again, I agree it's useless 😄